subprojects/docs/src/samples/userguide/dependencyManagement/troubleshooting/cache/changing/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,25 @@
+plugins {
+    `java-library`
+}
+
+repositories {
+    mavenCentral()
+    maven {
+        setUrl("https://repo.spring.io/snapshot/")
```

Comment:
there is better overload.
Use `maven(url=....)`

Reply:
:x: this should show:
```kotlin
repositories {
    maven {
        url = uri("...")
    }
}
```

@StefMa, yep, the overload can be handy. But let's stick to the more general and more close to Groovy `maven { url = uri("...") }` for user manual snippets. It also has the side effect of demonstrating how to create and assign a URI in the general case.